Jenkinsfile

```diff
@@ -5,7 +5,7 @@ def label = "treasury-${UUID.randomUUID().toString()}"
 podTemplate(label: label, containers: [
   containerTemplate(
     name: 'golang',
-    image: 'golang:1.12.6-alpine',
+    image: 'airhelp/golang:1.13-devops-builder',
```

Comment:
Won't this line https://github.com/AirHelp/dockerfiles/blob/master/golang/1.13-devops-builder/Dockerfile#L21 leave the GitHub token in the image?

Reply:
> Won't this line https://github.com/AirHelp/dockerfiles/blob/master/golang/1.13-devops-builder/Dockerfile#L21 leave the GitHub token in the image?

Yes, it will. This image is used for testing purposes and as a builder in the multistage build process.

Reply:
So it won't reach Docker Hub? I am asking because you wrote that 

> All Golang projects should use it.

Reply:
It's going to read the private docker hub images only for build and test jobs.